vm.go

```diff
@@ -243,7 +256,43 @@ func (vm *VM) EvaluateMulti(node ast.Node) (output map[string]string, err error)
 			err = fmt.Errorf("(CRASH) %v\n%s", r, debug.Stack())
 		}
 	}()
-	return evaluateMulti(node, vm.ext, vm.tla, vm.nativeFuncs, vm.globalBinding, vm.MaxStack, vm.importCache, vm.traceOut, vm.notifier, vm.StringOutput)
+
+	i, err := vm.buildInterpreter()
+	if err != nil {
+		return nil, err
+	}
+
+	return evaluateMulti(i, node, vm.tla, vm.StringOutput)
+}
+
+// Freeze builds the interpreter and makes it used by all subsequent evaluation calls.
+// This makes evaluation faster, however any future adjustments to native functions, global bindings etc. have no effect.
+// Also since the interpreter is no longer created separately for every evaluation, you shouldn't call more than one evaluation at the same time on such VM.
+func (vm *VM) Freeze() error {
```

Comment:
It is good enought now, but yes, it will be nice:
- If the Freeze can be called only once, (`freezed=true`)
- And if the all bindings/notifier/etc. setter will be panicking, if called after the `Freeze`.
  - I have no idea how many places must be modified.
  - It can be archive by wrapping a returning a `FreezedVM`, with only allowed methods, only `Evaluate`? 
  - or by direct modification of the library code.

Reply:
Yes! This PR can merged, of course.